examples/swarmion-with-cdk/backend/core/resources/stack.ts

```diff
@@ -13,6 +14,18 @@ export class CoreStack extends Stack {
   constructor(scope: Construct, id: string, props: StackProps & CoreProps) {
     super(scope, id, props);
 
-    new Health(this, 'Health');
+    const { stage } = props;
+
+    const coreApi = new RestApi(this, 'CoreApi', {
+      // the stage of the API is the same as the stage of the stack
+      description: `Core API - ${stage}`,
+      deployOptions: {
+        stageName: stage,
+      },
+    });
+
+    const healthResource = coreApi.root.addResource('health');
```

Comment:
should this ressource be added here or inside the Health function ?

Reply:
we could use [`resourceForPath`](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-apigateway.IResource.html#resourcewbrforwbrpathpath) method